tspec/base/posix/posix/errno.h.ts

```diff
@@ -3,8 +3,8 @@
 #
 # See doc/copyright/ for the full copyright terms.
 
-
-+EXP (extern) int errno ;
++FUNC (extern) int *__errno_location(void);
++DEFINE errno %% (*__errno_location ()) %%;
```

Comment:
Careful with these files - they're supposed to reflect the wording in each standard as closely as possible. If this differs from c89's definition for errno (`lvalue` vs. `extern`), you can bet the wording will be different.

Reply:
Perhaps it's an idea to put the definitions in new files? Although I have no knowledge of the standard and standard vs posix. And then it would need to select the right definition for the version that you want / os that you're using.

Reply:
Apologies, this is probably one of the most confusing parts about tendra. The idea is that it provides abstract definitions for each API - e.g. in tspec we say C89 defines `size_t` as "_an_ unsigned integer type", but not exactly which one.

Then your system headers provide the definition for exactly which kind of integer is used for `size_t`. osdep/ contains workarounds for when your system headers don't quite conform to a standard, and we also have the choice of providing our own alternative implementation entirely.

I'm not sure what might best suit errno there. But actually I don't understand what's causing `__errno_location` to get missed in the first place.

I'd be happy to set up the same OS and help with this, if you want me too :)